airflow/providers/google/cloud/operators/postgres_to_gcs.py

```diff
@@ -88,7 +90,7 @@ def convert_type(self, value, schema_type):
         Decimals are converted to floats. Times are converted to seconds.
         """
         if isinstance(value, (datetime.datetime, datetime.date)):
-            return time.mktime(value.timetuple())
+            return pendulum.parse(value.isoformat()).float_timestamp
```

Comment:
Holy in-efficiency batman!

We're doing from a datetime, to a string (a bit slow) parsing it _back_ to a datetime-like object (_VERY_ slow) to then call an attribute on it.

This should be

```
             return value.timestamp()
```

Also the comment should be clarified. "Times are converted to fractional-seconds"

Reply:
You're correct that this is inefficient. 
However, `value.timestamp` doesn't take timezone information into account.

Now that I think of it, it should have been written as  or some other way

```
return pendulum.timezone('UTC').convert(value).timestamp()
```

Reply:
@ashb WDYT? Should the commit be reverted and new one added?